gsl_core/admin.py

```diff
@@ -124,12 +130,20 @@ class PerimetreAdmin(AllPermsForStaffUser, admin.ModelAdmin):
         "region_id",
         "departement_id",
         "arrondissement_id",
+        "user_count",
     )
 
     def get_queryset(self, request):
         queryset = super().get_queryset(request)
         queryset = queryset.select_related("departement", "region")
+        queryset = queryset.annotate(user_count=Count("collegue"))
         return queryset
 
+    def user_count(self, obj):
+        return obj.user_count
+
+    user_count.admin_order_field = "user_count"
+    user_count.short_description = "# d'utilisateurs"
```

Comment:
Dans une interface en français, je mettrais plutôt l'abréviation "Nb" 

```suggestion
    user_count.short_description = "Nb d’utilisateurs"
```